### PR TITLE
feat: Update styling and restructure docs

### PR DESCRIPTION
**Pull request by ChialiTsai**
CSS update for color, font family, size and light touch up on mobile view. 
initial merge before reorganized information structure

### CODE REVIEW

**Comment by ChialiTsai** [comment]
Complete first batch of update, css style (code box needs assistance)
articles are now organized into new folders.

File: docusaurus.config.js [hunk 1]
```diff
@@ -53,16 +53,17 @@ const config = {
...
-          { to: "/api", label: "Tools", position: "right" },
-          { to: "/api", label: "Tutorials", position: "right" },
-          { to: "/api", label: "Governance", position: "right" },
-          { to: "/api", label: "API-to be delete", position: "right" },
+          // {
+          //   type: "doc",
+          //   docId: "Introduction",
+          //   position: "right",
+          //   label: "Overview",
+          // },
```
Comment (kalidiagne) [inline docusaurus.config.js:61]:
```suggestion
```

lets remove it if not needed

**Comment by kalidiagne** [comment]
@ChialiTsai  

LGTM! just a few tips

- If possible let's use less `!important` with more precise CSS selectors (https://stuffandnonsense.co.uk/archives/css_specificity_wars.html)
- Please install this Plugin to VSCode https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode, so you can easily format code, if needed I can show you how to use it!

File: src/css/custom.css [hunk 2]
```diff
@@ -153,5 +181,40 @@ main{
...
+@media (max-width: 996px){
+  .footer {
+    --ifm-footer-padding-horizontal: 1.5rem;
+}
+}
+
+
+/* snippet */
+
+.codeBlockLines_node_modules-\@docusaurus-theme-classic-lib-next-theme-CodeBlock-styles-module{
```
Comment (kalidiagne) [inline src/css/custom.css:205]:
```suggestion
code[class^="codeBlockLines"] {
```
we can simplify in this way

**Review by searchableguy — APPROVED** [review]

**Pull request merged**